Packages/com.unity.inputsystem/Tests/InputSystem/Monobehaviors/TrackedPoseDriverTests.cs

```diff
@@ -0,0 +1,155 @@
+using UnityEngine;
+using UnityEngine.TestTools;
+using NUnit.Framework;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine.SpatialTracking;
+using System.Linq;
+using UnityEngine.XR.PoseProvider;
+
+namespace UnityEngine.Experimental.Input.XR

```

Comment:
Move this file respectively to TrackedPoseDriver.cs. Kill namespace on tests. Just put them in the global namespace.

Reply:
Ok